app/assets/stylesheets/custom.scss

```diff
@@ -47,6 +47,28 @@ h3 {
   min-height: 50px;
 }
 
+#flash_error {
+  text-align: center;
```

Comment:
Properties should be ordered color, font-size, font-weight, margin-top, text-align